### ISSUE TITLE
can't make a link with a URL longer than 75 chars

### ISSUE THREAD

**Issue by durka** [lojban/jbovlaste#181]
[As seen here.](http://jbovlaste.lojban.org/comments.html?valsi=2386;natlangword=0;commentid=2159;definition=0) The word wrapping happens when the comment is posted, but the wiki formatting pass is later (during display), so links with long URLs have random `<br/>`s inserted in them.

**Comment by durka** [comment]
IMO, the real bug is hard wrapping the text at post time. That's a 100% display decision and we can do it HTML/CSS on the comment pages. This would be done by:

1. Add a `class="pinka"` attribute to the `<p>` tags that contain comments.
2. Add a style rule `p.pinka { width: 75ch; word-wrap: break-word }` to the page.

**Comment by djeik** [comment]
Yeah, hard-wrapping the text is a bad decision IMO. Plus, you get to avoid storing HTML in the backend.
